### PR TITLE
Update creating an image section

### PR DESCRIPTION
**Pull request by K-Meech**
This PR updates the 'Creating an image' section of the 'What is an image?' episode. This incorporates Nic Cade's comments.

@davecash75 - could you take a quick look and merge if you're happy?

### CODE REVIEW

File: episodes/what-is-an-image.md
Comment (davecash75) [inline episodes/what-is-an-image.md:164]: Maybe a slight tweak of "an excitation light source (e.g. a laser illuminates the sample, and this light is absorbed by (or excites a?) fluorescent label, causing it to emit light..."

**Review by davecash75 — COMMENTED** [review]
Only a minor suggestion. I'll leave it up to you whether you think it works or not.

**Review by davecash75 — APPROVED** [review]
Sorry, should have marked this approved, as I don't want to block the merge over my comment.

File: episodes/what-is-an-image.md
Comment (K-Meech) [reply episodes/what-is-an-image.md:164]: I'll add this small change and then merge - thanks!

**Pull request merged**
